iina/zh-Hans.lproj/QuickSettingViewController.strings

```diff
@@ -165,7 +165,7 @@
 "bzk-c2-LH5.label" = "音频";
 
 /* Class = "NSSegmentedCell"; cMu-YF-riv.ibShadowedLabels[0] = "Default"; ObjectID = "cMu-YF-riv"; */
-"cMu-YF-riv.ibShadowedLabels[0]" = "Default";
+"cMu-YF-riv.ibShadowedLabels[0]" = "默认";
```

Comment:
This may cause corresponding menu item not being selected correctly.

Please take a look at https://github.com/lhc70000/iina/blob/master/iina/MenuController.swift#L138 which should also be changed.